app/helpers/moment-add.js

```diff
@@ -0,0 +1,10 @@
+import helper from 'ember-helper';
+import moment from 'moment';
+
+export function momentAdd(params) {
+  const [dateAsMoment, addValue, addUnit, outputFormat] = params;
+  const newMoment = moment(dateAsMoment).clone().add(addValue, addUnit).format(outputFormat);
+  return newMoment;
+}
+
+export default helper.helper(momentAdd);
```

Comment:
this is a great helper, but you should name it something else, b/c the moment package already exposes a `moment-add` helper.